recipe/meta.yaml

```diff
@@ -13,11 +34,11 @@ source:
   url: https://releases.hashicorp.com/{{ name }}/{{ version }}/{{ name }}_{{ version }}_windows_amd64.zip  # [win64]
   url: https://releases.hashicorp.com/{{ name }}/{{ version }}/{{ name }}_{{ version }}_windows_386.zip  # [win32]
   fn: {{ name }}-{{ version }}.zip
-  sha256: d3999f80d3083ceee800ef8d0cf06dbe88d0449d4056203bd2f7491218b99d1d  # [osx]
-  sha256: f44d255d5a5126f95698cebc561a76f0824e9b53d391f84d0501d8ef36facbae  # [linux32]
-  sha256: 4782e4662de8effe49e97c50b1a1233c03c0026881f6c004144cc3b73f446ec5  # [linux64]
-  sha256: 1beb71cf1da5563d1dc05beb309b6a3bb2a884733bf8f9a9c3a851d148691712  # [win32]
-  sha256: 287581af4f30a19921bb6c3c4d80282dddae7e8ccced135366af83358e3f62a5  # [win64]
+  sha256: {{ sha256_osx }}  # [osx]
+  sha256: {{ sha256_linux32 }}  # [linux32]
+  sha256: {{ sha256_linux64 }}  # [linux64]
+  sha256: {{ sha256_win32 }}  # [win32]
+  sha256: {{ sha256_win64 }}  # [win64]
```

Comment:
These kind of changes are going to make the life of auto-tick bot hard

Reply:
The consul, vault, terraform, terraform-providers-* packages were not picked up by the auto-tick bot in its current iteration, so there is no downside for the moment. 

On the plus side, it makes updating these packages as simple as adding the sha256sum file directly from the upstream provider and changing the version number.